src/pages/invoices/common/hooks/useTaskColumns.ts

```diff
@@ -16,41 +16,127 @@ export function useTaskColumns() {
   const company = useCurrentCompany();
   const [columns, setColumns] = useState<string[]>([]);
 
+  const getColumnIndex = (
+    columnKey: string,
+    pdfColumns: string[],
+    currentColumns: string[]
+  ) => {
+    const columnIndex = pdfColumns.findIndex(
+      (variable) => variable === columnKey
+    );
+
+    const adjustedColumnIndex =
+      columnIndex > -1 ? columnIndex : currentColumns.length;
+
+    return adjustedColumnIndex;
+  };
+
```

Comment:
This looks really similar to `src/pages/invoices/common/hooks/useProductColumns.ts` getColumnIndex method. Can we reuse them?

Reply:
Yeah, it's the same. I added the comment to clarify the purpose of the function, and it's exported to be used within both hooks.